update(blog)!: Stay Safe but Stay Connected (#3050)
privacyguides/privacyguides.org: blog/posts/stay-safe-but-stay-connected.md

---
date:
    created: 2025-06-10T17:00:00Z
categories:
    - News
authors:
    - em

> description: In data privacy, we often talk about the dangers of data collection and exposed data. It can get overwhelming to learn more about all the information that is collected on us, especially at the beginning. But all or nothing approaches aren't solutions. We have to find a balance between data protection and the need for connection.

Author: Noted!👍 What about this instead?

Balancing data protection and online connection can be difficult. Nevertheless, it's an essential skill to be able to stay safe online while staying connected.

</div>

- [Improving privacy settings on X (formerly Twitter)](https://beconnected.esafety.gov.au/topic-library/social-media-apps/introduction-to-twitter/controlling-your-x-privacy-settings)
- [Improving privacy settings on Facebook](https://www.consumerreports.org/electronics-computers/privacy/facebook-privacy-settings-a1775535782/)
- [Improving privacy settings on Instagram](https://www.consumerreports.org/electronics-computers/privacy/instagram-privacy-settings-a3036233134/)
- [Improving privacy settings on Threads](https://www.myprivacy.blog/threads-privacy-setup-a-2025-technical-guide-for-users-under-25/)
- [Improving privacy settings on YouTube](https://www.groovypost.com/howto/manage-your-youtube-privacy-settings/)
- [Improving privacy settings on TikTok](https://www.maketecheasier.com/tiktok-privacy-settings/)
- [Improving privacy settings on Reddit](https://www.myprivacy.blog/reddit-privacy-guide-securing-your-presence-in-2025/)
- [Improving privacy settings on LinkedIn](https://www.presencesecure.com/linkedin-privacy-settings-guide/)

> Additionally, with [the proliferation of age verification](https://www.privacyguides.org/articles/2025/05/06/age-verification-wants-your-face/) regulations, more commercial platforms might soon not only require a legal name, but also require providing an official ID in order to continue using the platform, severely damaging online privacy for everyone.

Author: Ah yes, sorry I forgot for this one

In light of this, it might be a better idea to leave these intrusive and data exploitative platforms altogether, while deletion is still an option.

#### How to delete corporate social media accounts

> Deleting the accounts you no longer use is an essential habit to adopt in order to improve both privacy and security online.

Author: Ah yes! Excellent! I'll add it! 👍

Moreover, the [disappearing messages feature](https://support.signal.org/hc/en-us/articles/360007320771-Set-and-manage-disappearing-messages) will greatly help to reduce your data trail. Remember however that this will not prevent a malicious recipient from downloading or taking screenshots of the sensitive information you share with them. This is true for any service with disappearing messages. Only send sensitive information and photos to people you trust, even when using disappearing messages on encrypted platforms.

#### Other end-to-end encrypted messengers

While Signal might be the most popular and easier to use, there are many other instant messaging apps that offer solid end-to-end encryption features.

These will not even require a phone number to sign up. However, because they are less popular, it might be more difficult to find other users willing to use them with you to communicate.

> They also offer different features and experiences, that you might prefer, or not. If you prefer to use a different application, [Briar](https://briarproject.org/), [Cwtch](https://docs.cwtch.im/), [SimpleX](https://simplex.chat/), or [Threema](https://threema.com/en) are all good alternatives, depending on the features you are looking for.

Author: Yes, I agree. It will also age better if recommendations change overtime. I'll change it 👍

<div class="admonition warning" markdown>
<p class="admonition-title">Not all rooms are end-to-end encrypted</p>

Remember that not every room uses end-to-end encryption on Matrix. You can verify this in Element, from the detailed room information panel, by the "Encrypted" or "Not encrypted" badge under a room's name.

</div>

Again, because this open-source protocol is developed by a non-profit organization, the goal isn't to collect data on users. There is no incentive to monetize users' data, and because of this, choosing these better alternatives will greatly improve your data privacy compared to using commercial platforms.

#### End-to-end encrypted emails

For email communication, moving away from large providers such as Google and Microsoft can improve your communication's privacy immensely.

Large providers of free email services tend to collect a lot of data on their users, sometimes including the content of their communications. Moving to a service that uses end-to-end encryption and better practices around metadata offers significant benefits.

> Proton Mail and Tuta are two services implementing end-to-end encryption to protect the content of users' messages. For more details and options, you can look at our [Email Services providers](https://www.privacyguides.org/en/email/) recommendation page.

Reviewer: ```suggestion
Proton Mail and Tuta are two services implementing end-to-end encryption to protect the content of users' messages. For more details and options, you can look at our recommended [email services providers](https://www.privacyguides.org/en/email/) page.
```

> When using any of these services, you will however have to take into account to whom you are sending the email. For end-to-end encryption to properly protect data both in transit and at rest, the sender needs to use a service *compatible* with the recipient.

Reviewer: ```suggestion
Of course, when using any of these services, you will still have to take into account to whom you are sending the email. For end-to-end encryption to properly protect data both in transit and at rest, the sender needs to use a service *compatible* with the recipient.
```

There are methods and third-party applications that can be used to encrypt and decrypt a message from a provider that does not offer end-to-end encryption, but this adds additional steps casual users might not feel comfortable getting into.

That being said, if you use a Proton Mail address to email another Proton Mail user, by default all communication's content will be fully encrypted, and only accessible to you and the intended recipient. Similarly, if you use Tuta to email another Tuta user, communication will be fully end-to-end encrypted.

> Inter-provider communication will likely require additional steps however, or will not benefit from end-to-end encryption on the server where the message is sent. For example, when sending an email from a Proton Mail to a Gmail account, if no additional protections are used, the content of this message will still be accessible to Google on the recipient's side.

Reviewer: ```suggestion
Communications between different providers will likely require additional steps, however, or will not benefit from end-to-end encryption on the server where the message is sent. For example, when sending an email from a Proton Mail to a Gmail account, if no additional protections are used, the content of this message will still be accessible to Google on the recipient's side.
```

All of this to say, be mindful when using email communication to exchange sensitive data, and make sure to verify inter-service compatibility to stay fully protected.

> Even considering this, using a Tuta or Proton Mail account to communicate even with people using unencrypted services still have the benefit that your communication will at least be stored fully encrypted on *your* side.

Reviewer: ```suggestion
Even considering this, using a Tuta or Proton Mail account to communicate even with people using unencrypted services still has the benefit that your communication will at least be stored fully encrypted on *your* side.
```

> Even considering this, using a Tuta or Proton Mail account to communicated even with people using unencrypted services still have the benefit that your communication will at least be stored fully encrypted on *your* side.

Reviewer: ```suggestion
Even considering this, using a Tuta or Proton Mail account to communicate even with people using unencrypted services still have the benefit that your communication will at least be stored fully encrypted on *your* side.
```
